aiogram/types/message.py

```diff
@@ -673,6 +676,7 @@ class ContentType(helper.Helper):
     LOCATION = helper.ListItem()  # location
     VENUE = helper.ListItem()  # venue
     NEW_CHAT_MEMBERS = helper.ListItem()  # new_chat_member
+    LEFT_CHAT_MEMBER = helper.Item()  # left_chat_member
```

Comment:
In this place need to use `helper.ListItem()` instead of `helper.Item()`
`Item` returns string
`ListItem` returns list